### DISCUSSION TITLE
Schnell und einfach trading

### DISCUSSION THREAD

**Discussion by elnaz-gharoon** [SE-TINF22B2/G3-ApeRepublic#18] [User-Stories]
Als aktiver trader will ich schnell und einfach traden können, um Marktchancen in real-time nutzen zu können

**Comment by MatejEvc** [comment]
Ist real-time realistisch? (API Begrenzung)

**Reply by elnaz-gharoon** [reply]
wenn API verfügbar wäre, ja... muss noch während der Arbeit schauen
